coalib/bearlib/aspects/taste.py

```diff
@@ -87,5 +87,5 @@ def __set__(self, obj, value):
         """
         if self.name in obj.__dict__:
             raise AttributeError(
-                "can't set taste values of aspectclass instances")
+                'A taste value of aspectclass instances already exists.')
```

Comment:
`instance` should also be singular. 

Actually, the sentence reads much niceer like this `A 'taste' value for this aspectclass instance exists already.`. Would you agree?